vector/src/test/java/im/vector/app/features/location/live/map/UserLiveLocationViewStateMapperTest.kt

```diff
@@ -79,14 +84,16 @@ class UserLiveLocationViewStateMapperTest {
         val viewState = userLiveLocationViewStateMapper.map(summary)
 
         val expectedViewState = UserLiveLocationViewState(
-                userId = A_USER_ID,
+                matrixItem = MatrixItem.UserItem(id = A_USER_ID, displayName = "User 2", avatarUrl = ""),
```

Comment:
I think we will also need to mock the return of `getUser()` method for the `fakeSession`

Reply:
Exactly.

Reply:
Absolutely, done. Thanks for the help.